doc/design/dandiset-issues-tracking.md

```diff
@@ -0,0 +1,85 @@
+- The issue discussing this: https://github.com/dandi/dandi-archive/issues/863
+
+
+## Proposed Process
+
+**Overall**: Reuse existing Dandisets on GitHub (https://github.com/dandisets), and available for them GitHub's issue tracker.
+
+- [x] Enable Issue tracking per Dandiset.
+  - In the process of creating DataLad Dandisets on GitHub we already enable `Issues` functionality for all dandisets.
+- [ ] Initiate documentation in handbook on managing issues of the dandisets
+- [ ] Upon creation of a DataLad Dandiset on GitHub:
+  - Invite all Dandiset owners in Triage role to the dandiset. In Triage scope they can close, label, assign issues. The other actions (e.g. user management) would need to be implemented using [bot].
+  - Email them pointing to documentation (item above)
+- [ ] Setup https://github.com/dandisets [organization-wide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) [`.github/ISSUE_TEMPLATE`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) with different templates/types of issues to report
+- [ ] DLP: add a button "View/Report Issues" or "Issues" leading to issue Tracker (e.g. [Issues](https://github.com/dandisets/000108/issues) for 000108)
+  - Complimentary: dandi-archive could query/cache number of known open issues and include within that button, e.g. have it [(1) Issues](https://github.com/dandisets/000108/issues)
+  - Complimentary:  asset-level issues filing in UI context menu per file, would pre-fill in the asset path
+- [ ] Optional: provide consistent collection of labels to outline typical problems which might occur
+  - e.g., `validation-error`, `unable-to-load`, `io-error`, `missing-file`, `permission`, etc.
+  - ref: https://github.com/dandi/dandisets/issues/361
+
+### Alternatives considered
+
+#### GitHub, but not inviting users and operating using `@handles`
+
+Although sounded like a good idea at first, cumbersome to implement with consistent and/or desired behavior:
+
+- Dandiset owners would really have no power to do anything, like close the issue etc, unless we implement all such actions (assignment, labeling, etc) via a [bot].
+- To ensure that clicking on "New issue" would add needed `@handles` we would need to either
+  - add action to monitor new issues to enhance description with the `@handles`, or
+  - generate/update issue template per each dandiset with its own list of owners.
+
+### Changes on dandi-archive Backend
+
+- Provision configuration settings for
+  - GitHub organization where to create repository for a new dandiset
+  - GitHub token with sufficient permissions to create repository in that organization
+- If there is provisioned GitHub organization: Job creating a new dandiset entry would trigger a job to create a new repository.
```

Comment:
This section addresses the concerns I mentioned above, but there seems to be some contradiction between sections that we should resolve.

Reply:
With https://github.com/dandi/dandi-archive/pull/1595/commits/d8adab53e4f9c7875eeb3c1a8225dcb3fcd4d2ef this is now resolved.